weave-js/src/components/PagePanelComponents/Home/Browse3/pages/common/tabularListViews/operators.ts

```diff
@@ -58,79 +60,115 @@ export const operationConverter = (
       return null;
     }
     const val = parseFloat(item.value);
-    return {
-      $eq: [
-        {$convert: {input: {$getField: item.field}, to: 'double'}},
-        {$literal: val},
-      ],
-    };
+    if (FIELDS_NO_FLOAT_CONVERT.includes(item.field)) {
```

Comment:
Maybe more readable if we do something like this?
```
    let getField: any = {$getField: item.field};
    if (!FIELDS_NO_FLOAT_CONVERT.includes(item.field)) {
      getField = {
        $convert: {input: getField, to: 'double'},
      };
    }
    return {
      $eq: [getField, {$literal: val}],
    };
```

Reply:
Great idea, I took the liberty of making a helper, I think its worth it even though you now have to look in two places. 

Before: 
<img width="124" alt="Screenshot 2025-05-02 at 1 40 44 PM" src="https://github.com/user-attachments/assets/9a99c722-32a1-465c-805b-965679f22194" />

After:
<img width="130" alt="Screenshot 2025-05-02 at 1 40 50 PM" src="https://github.com/user-attachments/assets/b552ccd5-c151-4146-9897-612ee2bb83bc" />

Reply:
That's much nicer, thanks